cli/types/cypress.d.ts

```diff
@@ -1581,6 +1590,9 @@ declare namespace Cypress {
     select(value: string | string[], options?: Partial<SelectOptions>): Chainable<Subject>
 
     /**
+     * Deprecated - use `cy.http()` instead.
+     * @deprecated
```

Comment:
```suggestion
     * @deprecated Use `cy.http()` instead.
```

to match the others?